mne/viz/evoked.py

```diff
@@ -1498,9 +1502,31 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
         parameters will be passed to the line plot call of the corresponding
         condition, overriding defaults.
         E.g., if evokeds is a dict with the keys "Aud/L", "Aud/R",
-        "Vis/L", "Vis/R", `styles` can be `{"Aud/L":{"linewidth":1}}` to set
+        "Vis/L", "Vis/R", `styles` can be `{"Aud/L": {"linewidth": 1}}` to set
         the linewidth for "Aud/L" to 1. Note that HED ('/'-separated) tags are
         not supported.
+    cmap : None | str | tuple
+        If not None, plot evoked activity with colors from a color gradient
+        (indicated by a str referencing a matplotlib colormap - e.g., "viridis"
+        or "Reds").
+        If ``evokeds`` is a list and ``colors`` is `None`, the color will
+        depend on the list position. If ``colors`` is a list, it must contain
+        integers where the list positions correspond to ``evokeds``, and the
+        value corresponds to the position on the colorbar.
+        If ``evokeds`` is a dict, ``colors`` should be a dict mapping from
+        (potentially HED-style) condition tags to numbers corresponding to
+        rank order positions on the colorbar. E.g., ::
```

Comment:
aparat from the remark above - this is documentation for `cmap`, but it is mostly devoted to `colors` kwarg - maybe that should be there?

Reply:
I see what you mean. However, here's my rationale. `colors` has essentially two modes: a categorical mode, and a gradient mode. The gradient mode depends on `camp`.

Reply:
Yes, I get that, it is just a bit confusing, no big deal.